### PR TITLE
chore(documentation): adds breakpoint information

### PR DESCRIPTION
**Pull request by srambach**
Add documentation to the developer resources about how the breakpoints are used.

### CODE REVIEW

**Review by evwilkin — DISMISSED** [review]
LGTM - thanks!

File: packages/theme-patternfly-org/pages/global-css-variables.md [hunk 1]
```diff
@@ -42,6 +42,12 @@ Where...
 - `breakpoint` is a media query breakpoint such as `sm` for `$pf-global--breakpoint--xs`.
 - `pseudo-element` is either `before` or `after`.
 
+### Breakpoint variables and class suffixes
+
+PatternFly defines a number of standard breakpoints. These are always used as a min-width breakpoint; i.e. using the `-md` breakpoint would apply to everything at the `-md` width and wider.
```
Comment (mcoker) [inline packages/theme-patternfly-org/pages/global-css-variables.md:47]: Dunno if it's worth noting but the table is the exception to this.

```suggestion
PatternFly defines a number of standard breakpoints. These are always used as a `min-width` breakpoint; i.e. using the `-md` breakpoint would apply to everything at the `-md` width and wider.
```

File: packages/theme-patternfly-org/pages/global-css-variables.md [hunk 2]
```diff
@@ -42,6 +42,12 @@ Where...
...
+
+Breakpoint suffixes are used in utility classes and layouts as well as in many components. When available, `{-on-[breakpoint]}` will be shown as an available suffix in the usage section of the documentation. The breakpoint suffix is optional and if not specified, the base will be used. There are breakpoints for `sm`, `md`, `lg`, `xl`, and `2xl`. The values for the corresponding breakpoints are defined below in the global variables `—pf-global—breakpoint—[breakpoint]`.
```
Comment (mcoker) [inline packages/theme-patternfly-org/pages/global-css-variables.md:49]:
```suggestion
Breakpoint suffixes are used in utility classes and layouts as well as in many components as a way to apply class styles at a specified breakpoint. When available, `{-on-[breakpoint]}` will be shown as an available suffix for the class in the usage section of the documentation. The breakpoint suffix is optional and if not specified, the base class will be used. There are breakpoints for `sm`, `md`, `lg`, `xl`, and `2xl`, and the values for the corresponding breakpoints are defined below in the global variables `—pf-global—breakpoint—[breakpoint]`.
```

**Review by mcoker — APPROVED** [review]
Looks great! A couple of comments.

File: packages/theme-patternfly-org/pages/global-css-variables.md (hunk 1 above)
Comment (mattnolting) [reply packages/theme-patternfly-org/pages/global-css-variables.md:47]: @srambach @mcoker the table is the exception bc we're changing the default presentational behavior and breaking the built in semantics by making it `display: grid`. It doesn't make sense to assign `display: table` to a `table`, so this is an appropriate application of `min-width` media query as the table resumes its normal behavior at whatever breakpoint.

**Review by mcoker — APPROVED** [review]
🚀

**Pull request merged**